.github/workflows/staging-docker-testnet.yml

```diff
@@ -0,0 +1,138 @@
+name: Staging Testnet Pipeline
+
+on:
+  pull_request:
+    branches: [ master ]
+    types: [ opened, synchronize, closed ]
+
+env:
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+  ENV: "stg"
+  APP: "mdw-frontend-testnet"
+
+jobs:
+  main:
+    runs-on: ubuntu-latest    
+    name: Testnet Staging Pipeline
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      
+      - name: Set up Docker Buildx
+        id: buildx
+        # Use the action from the master, as we've seen some inconsistencies with @v1
+        # Issue: https://github.com/docker/build-push-action/issues/286
+        uses: docker/setup-buildx-action@master
+        # Only worked for us with this option on
+        with:
+          install: true
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          # Key is named differently to avoid collision
+          key: ${{ runner.os }}-${{ env.ENV }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-${{ env.ENV }}-buildx
+
+      - name: Log in to dockerhub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASS }}
+
+      - name: Extract metadata for docker
+        if: github.event_name == 'pull_request' && github.event.action == 'opened' || github.event.action == 'synchronize'
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: aeternity/mdw-frontend-testnet
```

Comment:
i don't think we have such thing

Reply:
you mean the image, if we dont have, will create one.